packages/playground/src/App.vue

```diff
@@ -248,11 +262,31 @@ function navigateToHome() {
   return $router.push(DashboardRoutes.Other.HomePage);
 }
 
+
 onMounted(async () => {
   await (window.$$appLoader || noop)();
   openProfile.value = true;
 });
 
+
+onMounted(async () => {
+  if (!localStorage.getItem(TIMEOUT_QUERY_KEY)) {
+    localStorage.setItem(TIMEOUT_QUERY_KEY, `${window.env.TIMEOUT}`);
+  } else {
+    window.env.TIMEOUT = +localStorage.getItem(TIMEOUT_QUERY_KEY)! * 1000;
+  }
+
+  if (!localStorage.getItem(TIMEOUT_DEPLOYMENT_KEY)) {
+    localStorage.setItem(TIMEOUT_DEPLOYMENT_KEY, `${gridStore.client.clientOptions.deploymentTimeoutMinutes}`);
+  } else {
+    const client = gridStore.client as GridClient;
+    if (client) {
+      client.clientOptions.deploymentTimeoutMinutes = +localStorage.getItem(TIMEOUT_DEPLOYMENT_KEY)! / 60;
+      await client._connect;
```

Comment:
You shouldn't do this, you need to call the connect method

```suggestion
      await client.connect();
```